.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+indent_size = 2
```

Comment:
It's the default indent size for files other than .json & .py. xml files (SVG ones) exported using Inkscape are using 2 as indent size